### DISCUSSION TITLE
Options for image block size

### DISCUSSION THREAD

**Discussion by sethduda** [GitbookIO/community#123] [Feature Requests]
### What's your feature/idea?

Would be nice if you could control the width/height of image blocks via the GitBook editor. You can align an image, but can't define how wide the image appears on the page.

### What's your desired outcome?

Would be nice to be able to drag the image block borders to adjust the displayed size, combined with existing alignment settings.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by aikoii** [comment]
Hi @sethduda,

Thanks for sharing this! Definitely, it would be great to have the option to resize images in the editor 🤞. 

Our current recommendation is that you upload each image at the size you want to use it, as our resizing options are currently quite limited, as you already know 😬. [TinyPNG](https://tinypng.com/) is a great option to reduce file size without affecting quality once you've got the physical image size sorted.

The maximum width we currently use in GitBook is 735px.

I hope this information is helpful!
Cheers 👋

**Comment by taszap** [comment]
Hey @sethduda we have shipped image resizing options. You can now resize images to full, large, medium or small size, see more info in our [changelog.](https://changelog.gitbook.com/april-2023/27-april-23-image-resizing-private-spaces-and-more) Let us know if you have any questions! 

<img width="569" alt="image" src="https://github.com/GitbookIO/community/assets/63643069/ab7dad56-5d5a-4ea7-9e17-bde65fd1ee12">.
